app/communication/views/banner.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import Q
+from rest_framework import status
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from app.common.mixins import ActionMixin
+from app.common.pagination import BasePagination
+from app.common.permissions import BasicViewPermission, IsHS
+from app.common.viewsets import BaseViewSet
+from app.communication.models.banner import Banner
+from app.communication.serializers.banner import BannerSerializer
+from app.util.utils import now
+
+
+class BannerViewSet(BaseViewSet, ActionMixin):
+    serializer_class = BannerSerializer
+    pagination_class = BasePagination
+    queryset = Banner.objects.all()
+    permission_classes = [IsHS]
+
+    @action(
+        detail=False,
+        methods=["get"],
+        url_path="visible",
+        permission_classes=[BasicViewPermission],
+    )
+    def get_visible_banner(self, request):
+        banner = Banner.objects.filter(
```

Comment:
Så du tidligere hadde en veldig fin `ModelManager` for `Banner` som kunne vært veldig fin å bruke her! 😄 (ikke et must)

Reply:
Fjernet det siden jeg fikk beskjed om å ikke bruke det for badges. Litt usikker på om det blir samme for bannere i dette tilfellet. Som jeg skjønte det er det et problem med hvordan managere oppdateres med databasen. @olros hva var grunnen til å ikke bruke for badges? Blir det samme her?

Reply:
Ja det var noe med når tidsfunksjonene parses - vi hadde problemer med det ift events. Tror du bare kan droppe det her egentlig